# can't record plots

    Code
      verify_output(tempfile(), plot(1:10))
    Condition
      Error in `FUN()`:
      ! Plots are not supported.

